docs.json

```diff
@@ -457,17 +457,21 @@
                     },
                     {
                         "title": "LoRaWAN",
-                        "intro": {
```

Comment:
For consistency with the other sections, I would like the section overview page to have general concepts about the technology and be followed by the API references. Because of this, I don't think we should start the section with the API reference.

Reply:
Other sections do start the page with API docs. And I was told that there is no requirement to have an intro. We could re-write the LoRaWANInterface section to reflect a bit about technology. But then why there is a separate Technology section dedicated only for this which you are suggesting here ?

Reply:
@hasnainvirk 

Every user API section (platform, drivers, RTOS, network socket, network interfaces, bluetooth, security and storage) starts with an overview page instead of going straight to the user API class reference. The individual class references within those sections don't require their own overview pages. Because LoRa is its own section, it requires an overview page. If it were nestled under network interfaces or something, it wouldn't require an overview page because there's already one for the section.

The section overview and technology pages are similar in that they both include reference material about our APIs and technology. However, our section overview pages include general concepts about their sections. The [platform page](https://os-doc-builder.test.mbed.com/docs/development/reference/platform.html) is a really good example of this. The technology pages are only for our stack technologies and include a deep-dive view of the technology's technical details and architecture.